From 1d73075be2e9fdddbff374ac500abb8183c49f22 Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Wed, 29 Feb 2012 14:09:53 +0800
Subject: ENGR00175692-02 [MX6DL] SD3.0: can not recognize SD3.0 cards on some
 boards.

Improve SD3.0 compatibility: change DSE to 34ohm for 200Mhz.

Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6dl.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h b/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
index b32cc00..6d0a53b 100644
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6dl.h
@@ -48,7 +48,7 @@
 
 #define MX6DL_USDHC_PAD_CTRL_200MHZ (PAD_CTL_PKE | PAD_CTL_PUE |	\
 		PAD_CTL_PUS_47K_UP  | PAD_CTL_SPEED_HIGH |		\
-		PAD_CTL_DSE_40ohm   | PAD_CTL_SRE_FAST   | PAD_CTL_HYS)
+		PAD_CTL_DSE_34ohm   | PAD_CTL_SRE_FAST   | PAD_CTL_HYS)
 
 #define MX6DL_ENET_PAD_CTRL (PAD_CTL_PKE | PAD_CTL_PUE  |		\
 		PAD_CTL_PUS_100K_UP | PAD_CTL_SPEED_MED |		\
-- 
1.7.9.5

